src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -161,14 +161,21 @@ export default App.extend({
 
     channel.request('subscribe', this.collection.models, { filters: { [filterKey]: this.filters } });
 
-    this.listenTo(channel, 'message', (message, model) => {
+    this.listenTo(channel, 'message', ({ category, resource, payload }) => {
+      if (!category) return;
```

Comment:
In my local dev environment I'm getting a ws notification that contains a lambda error. When that happens this `listenTo` is called with `category`, `resource`, and `payload` as `undefined`.

I don't know if this is something that could happen in production or not.

<img width="833" alt="Screenshot 2025-02-24 at 4 50 48 PM" src="https://github.com/user-attachments/assets/5ab13ce5-f8ba-42e2-86b7-2d98b5c96436" />

Reply:
we shouldn't need the `if` here.  should be able to handle those errors prior to triggering the message

Reply:
I'm looking at that currently. Would be in the next commit. I have a `event.data.category` check add to the `onMessage()` [here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/services/ws.js#L101C3-L101C12). 

So at the top of that function it would be:

```js
if (!event.data || !event.data.category) return;
```

But still need to test it locally and think about it a little more.